ktor-server/ktor-server-plugins/ktor-server-forwarded-header/jvmAndNix/src/io/ktor/server/plugins/ForwardHeaderSupport.kt

```diff
@@ -104,60 +77,151 @@ public object XForwardedHeaderSupport :
         /**
          * Port X-header names. Default is `X-Forwarded-Port`
          */
-        @OptIn(InternalAPI::class)
-        @PublicAPICandidate("2.0.0")
-        internal val portHeaders: ArrayList<String> = arrayListOf("X-Forwarded-Port")
+        public val portHeaders: ArrayList<String> = arrayListOf("X-Forwarded-Port")
```

Comment:
could you tell me why ArrayList is used here?

Reply:
no changes here since the previous version. I'll update it to a mutable list